src/components/FocusBoundary.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+
+function isChildOf(target, child) {
```

Comment:
Just wondering, did you avoid `node.contains` intentionally? https://developer.mozilla.org/en-US/docs/Web/API/Node/contains

Reply:
😱 

Nope, never heard of it! Damn... 😅